### PR TITLE
Feature/change report

### PR DESCRIPTION
**Pull request by analuizamtg**
**What is the purpose of this pull request?**
- To add dropdown so that the user can change report display

image

(For some reason power bi is not displaying the numbers today, but that's an unrelated problem)

### CODE REVIEW

File: react/index.js [hunk 1]
```diff
@@ -31,43 +15,20 @@ class App extends Component {
...
-            <Spinner />
-          </div>
-        ) : error ? (
-          <div className="ma10">
-            <EmptyState title={error.title || 'Someting went wrong... 😢'}>
-              <span>{error.message || `Generic error\n${error}`}</span>
-            </EmptyState>
+        {error ? (
+          <div className="ma8">
+            <EmptyState title={error.title || 'Someting went wrong... 😢'} />
```
Comment (kevinch) [inline react/index.js]: Hard coded text?

File: react/locales/en.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
   "subscriptions.title": "Subscriptions",
   "report.notFound": "Report not found",
-  "report.today-vs-yesterday": "Yesterday vs Today",
-  "report.today-vs-lastweek": "Today vs Last week",
+  "report.today-vs-yesterday": "Yesterday vs today",
+  "report.today-vs-lastweek": "Today vs last week",
```
Comment (Caio-Batista) [inline react/locales/en.json]: I think this should be still capital letter.

File: react/locales/es.json [hunk 3]
```diff
@@ -1,8 +1,8 @@
 {
   "subscriptions.title": "Suscripciones",
   "report.notFound": "Informe no encontrado",
-  "report.today-vs-yesterday": "Hoy x Ayer",
-  "report.today-vs-lastweek": "Hoy x Semana pasada",
+  "report.today-vs-yesterday": "Hoy x ayer",
```
Comment (Caio-Batista) [inline react/locales/es.json]: ditto

File: react/locales/pt.json [hunk 4]
```diff
@@ -1,8 +1,8 @@
 {
   "subscriptions.title": "Assinaturas",
   "report.notFound": "Relatório não encontrado",
-  "report.today-vs-yesterday": "Ontem x Hoje",
-  "report.today-vs-lastweek": "Hoje x Semana passada",
+  "report.today-vs-yesterday": "Ontem x hoje",
```
Comment (Caio-Batista) [inline react/locales/pt.json]: ditto

File: react/report.js [hunk 5]
```diff
@@ -1,34 +1,51 @@
...
+      },
+      loading: false,
     }
+  }
+
+  getToken = () => {
+    const { setError } = this.props
+    const { embedConfig } = this.state
+    this.toggleLoading()
     let accountName = window.location.origin.split('.myvtex')[0]
```
Comment (kevinch) [inline react/report.js]: Account name can be obtained via the context.

File: react/report.js [hunk 6]
```diff
@@ -38,31 +55,68 @@ export default class _Report extends Component {
       )
   }
 
+  translateOptions() {
+    return reportOptions.map(option => {
```
Comment (kevinch) [inline react/report.js]: Nitpick:
```
return reportOptions.map(option => ({
        label: ...,
        value: ...
    })
)
```

**Pull request merged**